### PR TITLE
Fix handling for readers when available_cash is not present.

### PR DESCRIPTION
**Pull request by m-d-brown**
This is required for current Vanguard retirement OFX files.

### CODE REVIEW

**Comment by redstreet** [comment]
Thanks for the PR. Let me run my local tests on it, probably this weekend.

Curious, what do you mean by "retirement"? Employer sponsored? Individual? 401k? Roth?

**Comment by m-d-brown** [comment]
I meant 401(k) specifically. I've updated the commit message to be more clear.

Thanks for sharing these importers. I've found them much simpler and more accessible than alternatives. I have some other changes underway to extend the OFX importer for Vanguard 401(k) accounts: https://github.com/m-d-brown/beancount_reds_importers/commits/mdbrown-vanguard401k. Would you be interested in a regression test (https://github.com/m-d-brown/beancount_reds_importers/commit/99a31bc396a93a4088e6aad57093b8427374cf2b) as a start?

File: beancount_reds_importers/libreader/ofxreader.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def get_balance_positions(self):
             yield pos
 
     def get_available_cash(self):
+        if not hasattr(self.ofx_account.statement, 'available_cash'):
+            return None
         return self.ofx_account.statement.available_cash
```
Comment (redstreet) [inline beancount_reds_importers/libreader/ofxreader.py:61]: return getattr(self.ofx_account.statement, 'available_cash', None)  # pythonic

**Review by redstreet — CHANGES_REQUESTED** [review]
Looks great, just the one small comment (I deleted my previous comments).

File: beancount_reds_importers/libreader/ofxreader.py (hunk 1 above)
Comment (m-d-brown) [reply beancount_reds_importers/libreader/ofxreader.py]: That's better. Thanks. Done.

**Comment by redstreet** [comment]
Awesome, glad you found these useful! Just in case you haven't come across it, here are the [companion articles](https://reds-rants.netlify.app/personal-finance/the-five-minute-ledger-update/).

Regression tests would be awesome! I currently test across hundreds of my personal files, and haven't had a chance to create synthetic tests yet. Would love to have a PR on that commit you pointed to.

Another PR on the narration commit you have is also appreciated.

Thank you for the contributions! :)

**Pull request merged**